### PR TITLE
story/TYTA-40/implement-authentification-middleware

### PR DESCRIPTION
**Pull request by BrystoQ**
##Description
Add Authentification middleware, but not attribute to routes. Wait meeting to talk about it

### CODE REVIEW

File: src/middleware/authMiddleware.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const jwtKey = process.env.JWT_KEY;
+
+exports.verifyToken = (req, res, next) => {
+  const token = req.headers.authorization;
+
+  if (token !== undefined) {
+    jwt.verify(token, jwtKey, (error) => {
+      if (error) {
+        console.log(error);
```
Comment (remirubis) [inline src/middleware/authMiddleware.js]: Please remove console logs before pushing 😃 
```suggestion
```

File: src/routes/userRoute.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 const userCtrl = require('../controllers/userCtrl');
+// const authMiddleware = require('../middleware/authMiddleware');
```
Comment (remirubis) [inline src/routes/userRoute.js:2]: Why it's commented for now ?

**Review by remirubis — CHANGES_REQUESTED** [review]
Little question and one changes

File: src/routes/userRoute.js (hunk 2 above)
Comment (BrystoQ) [reply src/routes/userRoute.js:2]: Was for testing, and i think will implement here to so i didn't remove it. we can if you want

Comment (remirubis) [reply src/routes/userRoute.js:2]: Okay you can let it commented in that case :)

**Review by remirubis — APPROVED** [review]
LGTM! 🚀

**Review by FabienHP — APPROVED** [review]
LGTM

**Pull request merged**